force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1785,6 +1785,14 @@
         <shortDescription>Status Reason to indicates Elevate Service is in progress.</shortDescription>
         <value>Pending Elevate Process</value>
     </labels>
+    <labels>
+        <fullName>RD2_DayOfMonth</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Day of Month label for field to be shown in rd2RecurringDonation component</shortDescription>
+        <value>Day of Month</value>
```

Comment:
If possible, could we instead use the Label of the Day_of_Month__c field on Recurring Donation?

Reply:
Good suggestion. Implemented